projects/Canary.Tests/ConnectathonControllerTests.cs

```diff
@@ -31,7 +32,12 @@ public void BFDRConnectathon()
             var response = controller.IndexBFDR();
             var result = controller.IndexBFDR();
             BirthRecord romeroConnectathon = result[0];
-            Assert.Equal(JsonConvert.SerializeObject(br), JsonConvert.SerializeObject(romeroConnectathon));
+            string timeZoneOffset = TimeZoneInfo.Local.GetUtcOffset(new DateTime(2000, 1, 1)).ToString()[..6];
```

Comment:
add comments explaining why the local time is expected since it came from IJE data